cypress/integration/login.spec.ts

```diff
@@ -18,6 +18,22 @@ describe('Login process', () => {
         cy.get('[data-test="username"]').should('be.visible');
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
+
+    it.only('clicking on user avatar takes you to My Page', () => {
+        cy.visit('/login');
+        cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
+        cy.get('input[aria-label="Password"]').type('password');
+        cy.get('input[aria-label="Log in"]')
+            .should('be.visible')
+            .click();
+        cy.url().should('match', /\/$/);
+        cy.get('[data-test="username"').should('be.visible');
+        cy.get('[data-test="user-avatar"]')
+            .should('be.visible')
```

Comment:
This line is not necessary. The test before this one already checks that it's visible, and here we don't really care about that, but are instead just interested in making sure that when the link is clicked, it leads to the right URL.